lib/travis/api/build/app.rb

```diff
@@ -20,16 +20,17 @@ class App < Sinatra::Base
         end
 
         configure do
-          use Sentry unless Travis::Build.config.sentry_dsn.empty?
-          use Metriks unless Travis::Build.config.librato.email.empty? ||
-                             Travis::Build.config.librato.token.empty? ||
-                             Travis::Build.config.librato.source.empty?
+          use Sentry unless Travis::Build.config.sentry_dsn.nil?
+          use Metriks unless Travis::Build.config.librato.email.nil? ||
+                             Travis::Build.config.librato.token.nil? ||
+                             Travis::Build.config.librato.source.nil?
```

Comment:
> If any of these things is nil, could it be that the corresponding key is missing in the config?

Yup, that's generally what's happening here. They're not getting set because the customer doesn't have Librato setup. 



> Coerce config values to Strings before calling `#empty?`; e.g., `Travis::Build.config.sentry_dsn.to_s.empty?`

I think this is probably the most direct approach. Overall I'm not fond of the fact that we don't know if and when we have config and how it could be set to `""` or `nil` but that's a much bigger problem. Normalizing that could be a good maintenance project for `travis-config`, as it feels like there are possible bugs waiting around in projects due to `nil`/`empty` checking. 

Thanks for the review on this stuff!